integrated_channels/degreed2/exporters/content_metadata.py

```diff
@@ -153,3 +162,46 @@ def transform_obsolete(self, content_metadata_item):  # pylint: disable=unused-a
         Always set obsolete to false to fix externally deleted courses.
         """
         return False
+
+    def transform_price(self, content_metadata_item):
+        """
+        Return the current course run's price.
+        """
+        price = 0
+
+        if self.enterprise_configuration.show_course_price:
+            advertised_course_run = get_advertised_course_run(content_metadata_item)
+            if advertised_course_run and 'first_enrollable_paid_seat_price' in advertised_course_run:
+                price = advertised_course_run.get('first_enrollable_paid_seat_price') or 0
+            else:
+                for course_run in content_metadata_item.get('course_runs', []):
+                    if 'first_enrollable_paid_seat_price' in course_run and is_course_run_active(course_run):
+                        price = course_run.get('first_enrollable_paid_seat_price') or 0
+                        break
+
+        return price
```

Comment:
question: if Degreed is expecting the following payload

```
payload = { "data": { "attributes": {
            "external-id": "string",
            "provider-code": "string",
            "title": "string",
            "summary": "string",
            "publish-date": "2023-07-19T17:28:24.871Z",
            "url": "string",
            "obsolete": True,
            "image-url": "string",
            "language": "string",
            "duration": 0,
            "duration-type": "string",
            "cost-units": 0,
            "cost-unit-type": "string",
            "format": "string",
            "difficulty": "string",
            "video-url": "string",
            "visibility-groups": [
                {
                    "id": "string",
                    "type": "groups"
                }
            ]
        } } }
```

is this the right data structure to return?